src/api/stats/avax/getPangolinV2DualApys.js

```diff
@@ -0,0 +1,145 @@
+const BigNumber = require('bignumber.js');
+const { MultiCall } = require('eth-multicall');
+const { avaxWeb3: web3, multicallAddress } = require('../../../utils/web3');
+
+const MasterChef = require('../../../abis/avax/PangolinChef.json');
+const Rewarder = require('../../../abis/avax/PangolinRewarderViaMultiplier.json');
+const ERC20 = require('../../../abis/ERC20.json');
+const fetchPrice = require('../../../utils/fetchPrice');
+const pools = require('../../../data/avax/pangolinV2DualLpPools.json');
+const { BASE_HPY, AVAX_CHAIN_ID } = require('../../../constants');
+const { getTradingFeeAprSushi } = require('../../../utils/getTradingFeeApr');
+import { getFarmWithTradingFeesApy } from '../../../utils/getFarmWithTradingFeesApy';
+const { pangolinClient } = require('../../../apollo/client');
+const { compound } = require('../../../utils/compound');
+
+const masterchef = '0x1f806f7C8dED893fd3caE279191ad7Aa3798E928';
+const oracleIdA = 'PNG';
+const oracleA = 'tokens';
+const DECIMALSA = '1e18';
+
+const secondsPerBlock = 1;
+const secondsPerYear = 31536000;
+
+const liquidityProviderFee = 0.0025;
+const beefyPerformanceFee = 0.045;
+const shareAfterBeefyPerformanceFee = 1 - beefyPerformanceFee;
+
+const getPangolinV2DualApys = async () => {
+  let apys = {};
+  let apyBreakdowns = {};
+
+  const tokenPriceA = await fetchPrice({ oracle: oracleA, id: oracleIdA });
+  const { rewardPerSecond, totalAllocPoint } = await getMasterChefData();
+  const { balances, allocPoints, rewarders } = await getPoolsData(pools);
+
+  const pairAddresses = pools.map(pool => pool.address);
+  const tradingAprs = await getTradingFeeAprSushi(
+    pangolinClient,
+    pairAddresses,
+    liquidityProviderFee
+  );
+
+  for (let i = 0; i < pools.length; i++) {
+    const pool = pools[i];
+
+    const lpPrice = await fetchPrice({ oracle: 'lps', id: pool.name });
+    const totalStakedInUsd = balances[i].times(lpPrice).dividedBy('1e18');
+
+    const poolBlockRewards = rewardPerSecond.times(allocPoints[i]).dividedBy(totalAllocPoint);
+    const yearlyRewards = poolBlockRewards.dividedBy(secondsPerBlock).times(secondsPerYear);
+    const yearlyRewardsAInUsd = yearlyRewards.times(tokenPriceA).dividedBy(DECIMALSA);
+    const yearlyRewardsBInUsd = await (async () => {
+      if (rewarders[i] === '0x0000000000000000000000000000000000000000') {
+        return 0;
+      } else {
+        const tokenPriceB = await fetchPrice({ oracle: pool.oracleB, id: pool.oracleIdB });
+        const rewarderContract = new web3.eth.Contract(Rewarder, rewarders[i]);
+        const tokenBMultiplier = new BigNumber(
+          await rewarderContract.methods.getRewardMultipliers().call()
```

Comment:
this is not good place for another eth call, it should be included in getPoolsData multicall.
otherwise if it grows to 100 pools, we will have 100 calls here, instead of 1 with multicall.

i understand there is no rewarderAddress at the time of first call, so there are 2 ways:
1) if rewarder is immutable in MC, we can hardcode it in pools.json and get multiplier in multicall.
2) if its mutable, we can get rewarder in multicall, and then make another multicall inside getPoolsData to get list of multipliers, then we return { balances, allocPoints, rewardMultiplers } from getPoolsData, we dont really need to return rewarder from there. could return 0 if no rewarder

also there can be multiple rewards, not sure if it ever will